sentry-samples/sentry-samples-spring-boot/build.gradle.kts

```diff
@@ -0,0 +1,40 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+plugins {
+    id("org.springframework.boot") version "2.3.3.RELEASE"
+    id("io.spring.dependency-management") version "1.0.10.RELEASE"
```

Comment:
would be possible to reuse the `Config` class like the SDK itself? they are probably the same groups/modules/versions

Reply:
this `build.gradle.kts` is kept as close as possible to one generated using https://start.spring.io - I think if we don't touch it much it will be easier to understand for users what exactly needs to be set up to use Sentry.

Reply:
I get your point, but the thing is, the samples on this repo are to easier the development/testing/debugging, not really customer samples, they still could learn from it, but that's not the main reason, we have eg https://github.com/getsentry/examples/tree/master/java
we'd be updating the examples after publishing these new integrations and there we'd keep as close as possible.

Reply:
Ok, got it. Would you also like to put all the dependencies from sample application to `Config` class?

Reply:
for me, it only makes sense the ones that are also used in the SDK, the dependencies of the samples that are only used in the samples, its fine to be hardcoded.
if it happens that a dependency of a sample is hardcoded but still share a version that is in the config file, then I'd do a string template like `"dependency$Config.sharedVersion"` so we know that if we update something, it's done only in one place, this is just a nit about DRY though.